spring-cloud-dataflow-server-core/src/test/resources/schemas/drop-table-schema-mariadb.sql

```diff
@@ -1,68 +1,50 @@
-ALTER TABLE IF EXISTS BATCH_STEP_EXECUTION DROP FOREIGN KEY IF EXISTS  JOB_EXEC_STEP_FK;
-ALTER TABLE IF EXISTS BATCH_JOB_EXECUTION DROP FOREIGN KEY IF EXISTS  JOB_INST_EXEC_FK;
-ALTER TABLE IF EXISTS BATCH_JOB_EXECUTION_CONTEXT DROP FOREIGN KEY IF EXISTS  JOB_EXEC_CTX_FK;
-
-ALTER TABLE IF EXISTS TASK_EXECUTION_PARAMS DROP FOREIGN KEY IF EXISTS TASK_EXEC_PARAMS_FK;
-ALTER TABLE IF EXISTS TASK_TASK_BATCH DROP FOREIGN KEY IF EXISTS TASK_EXEC_BATCH_FK;
-ALTER TABLE IF EXISTS TASK_EXECUTION_METADATA DROP FOREIGN KEY IF EXISTS TASK_METADATA_FK;
-
-ALTER TABLE IF EXISTS BOOT3_BATCH_JOB_EXECUTION DROP FOREIGN KEY IF EXISTS  BOOT3_JOB_INST_EXEC_FK;
-ALTER TABLE IF EXISTS BOOT3_BATCH_STEP_EXECUTION DROP FOREIGN KEY IF EXISTS  BOOT3_JOB_EXEC_STEP_FK;
-ALTER TABLE IF EXISTS BOOT3_TASK_EXECUTION_METADATA DROP FOREIGN KEY IF EXISTS  BOOT3_TASK_METADATA_FK;
-ALTER TABLE IF EXISTS BOOT3_BATCH_STEP_EXECUTION_CONTEXT DROP FOREIGN KEY IF EXISTS  BOOT3_STEP_EXEC_CTX_FK;
-ALTER TABLE IF EXISTS BOOT3_TASK_EXECUTION_PARAMS DROP FOREIGN KEY BOOT3_TASK_EXEC_PARAMS_FK;
-ALTER TABLE IF EXISTS BOOT3_TASK_TASK_BATCH DROP FOREIGN KEY BOOT3_TASK_EXEC_BATCH_FK;
-
-DROP TABLE IF EXISTS app_registration;
-DROP TABLE IF EXISTS task_deployment;
-DROP TABLE IF EXISTS audit_records;
-DROP TABLE IF EXISTS stream_definitions;
-DROP TABLE IF EXISTS task_definitions;
-DROP TABLE IF EXISTS TASK_EXECUTION_PARAMS;
-DROP TABLE IF EXISTS TASK_TASK_BATCH;
-DROP TABLE IF EXISTS TASK_SEQ;
-DROP TABLE IF EXISTS TASK_LOCK;
-DROP TABLE IF EXISTS TASK_EXECUTION;
-DROP TABLE IF EXISTS TASK_EXECUTION_METADATA;
-
-DROP TABLE IF EXISTS BATCH_JOB_INSTANCE;
-DROP TABLE IF EXISTS BATCH_JOB_EXECUTION;
-DROP TABLE IF EXISTS BATCH_JOB_EXECUTION_PARAMS;
-DROP TABLE IF EXISTS BATCH_STEP_EXECUTION;
-DROP TABLE IF EXISTS BATCH_STEP_EXECUTION_CONTEXT;
-DROP TABLE IF EXISTS BATCH_JOB_EXECUTION_CONTEXT;
-DROP TABLE IF EXISTS BATCH_STEP_EXECUTION_SEQ;
-DROP TABLE IF EXISTS BATCH_JOB_EXECUTION_SEQ;
-DROP TABLE IF EXISTS BATCH_JOB_EXECUTION_SEQ;
-DROP TABLE IF EXISTS BATCH_JOB_SEQ;
-DROP TABLE IF EXISTS TASK_EXECUTION_METADATA_SEQ;
-
 DROP VIEW IF EXISTS AGGREGATE_TASK_EXECUTION;
 DROP VIEW IF EXISTS AGGREGATE_TASK_EXECUTION_PARAMS;
 DROP VIEW IF EXISTS AGGREGATE_JOB_EXECUTION;
 DROP VIEW IF EXISTS AGGREGATE_JOB_INSTANCE;
 DROP VIEW IF EXISTS AGGREGATE_TASK_BATCH;
 DROP VIEW IF EXISTS AGGREGATE_STEP_EXECUTION;
 
-DROP TABLE IF EXISTS BOOT3_BATCH_JOB_INSTANCE;
-DROP TABLE IF EXISTS BOOT3_TASK_EXECUTION;
-DROP TABLE IF EXISTS BOOT3_TASK_EXECUTION_PARAMS;
-DROP TABLE IF EXISTS BOOT3_BATCH_STEP_EXECUTION;
-DROP TABLE IF EXISTS BOOT3_BATCH_STEP_EXECUTION_CONTEXT;
-DROP TABLE IF EXISTS BOOT3_BATCH_JOB_EXECUTION_CONTEXT;
-DROP TABLE IF EXISTS BOOT3_TASK_TASK_BATCH;
-DROP TABLE IF EXISTS BOOT3_TASK_LOCK;
-DROP TABLE IF EXISTS BOOT3_TASK_EXECUTION_METADATA;
-DROP TABLE IF EXISTS BOOT3_BATCH_JOB_EXECUTION;
-DROP TABLE IF EXISTS BOOT3_BATCH_JOB_EXECUTION_PARAMS;
+DROP TABLE IF EXISTS app_registration CASCADE;
```

Comment:
What is going on in this file? A self-review summary comment would help me here. :)

Reply:
This was only case using MariaDB above 10.6
We change to drop cascade because it was complaining of foreign keys. 
We had to fix the sequence.